test/groups/special_orthogonal.jl

```diff
@@ -90,7 +90,7 @@ include("group_utils.jl")
             inverse_retraction_methods = inverse_retraction_methods,
             exp_log_atol_multiplier = 20,
             retraction_atol_multiplier = 12,
-            is_tangent_atol_multiplier = 1,
+            is_tangent_atol_multiplier = 1.2,
```

Comment:
Any ideas why the change in tolerance is necessary? I'm wondering if a different default is being hit now than before.

Reply:
I have no idea. It's a very small change so maybe there is no reason to worry about it?

Reply:
Probably!